Zend/zend_API.h

```diff
@@ -246,14 +246,14 @@ ZEND_API int zend_copy_parameters_array(int param_count, zval *argument_array);
 #define zend_get_parameters_array_ex(param_count, argument_array) \
 	_zend_get_parameters_array_ex(param_count, argument_array)
 #define zend_parse_parameters_none() \
-	(EXPECTED(ZEND_NUM_ARGS() == 0) ? SUCCESS : zend_wrong_parameters_none_error())
-#define zend_parse_parameters_none_throw() \
 	(EXPECTED(ZEND_NUM_ARGS() == 0) ? SUCCESS : zend_wrong_parameters_none_exception())
+#define zend_parse_parameters_none_throw() \
+	zend_parse_parameters_none()
 
 /* Parameter parsing API -- andrei */
 
+#define ZEND_PARSE_PARAMS_THROW 0 /* No longer used, zpp always uses exceptions */
```

Comment:
I think you should remove this define, so that it is more apparent that if this is used, the code no longer functions according to the intention.

Reply:
It still works as intended though -- it's just no longer necessary to specify it explicitly. I think especially in conjunction with `zend_parse_parameters_throw` and `ZEND_PARSE_PARAMS_THROW`, it would be best to preserve these functions/macros, so as to avoid unnecessary churn for extensions that already opt-in to throwing. Otherwise each extension will have to define these for compatibility.